.github/workflows/release.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     tags:
       - v*.*.*
+  workflow_dispatch:
```

Comment:
I guess this isn't related, right? Could you remove it?

Reply:
Nope.  It's there so I could test more easily.  I'll remove it once everything else is ready to merge.

Reply:
Done